src/components/annotations3d/attributes.js

```diff
@@ -75,13 +75,13 @@ module.exports = {
     arrowwidth: annAtts.arrowwidth,
     standoff: annAtts.standoff,
     hovertext: annAtts.hovertext,
-    hoverlabel: annAtts.hoverlabel
+    hoverlabel: annAtts.hoverlabel,
+    captureevents: annAtts.captureevents
```

Comment:
- commit 8b8a088 generalises `drawRaw`
- commit 1d5606a add `subplotId` to the `ploltly_clickannotation` event data + :lock: it down.